baseline/OSPS-VM.yaml

```
@@ -158,6 +165,17 @@ controls:
           - 2.1
           - 2.4
           - 2.6
+       - reference-id: SSDF
+        identifiers:
+          - PO4.1
+          - RV2.1
+          - RV2.2
```

Small indentation nit:
```suggestion
         identifiers:
            - PO4.1
            - RV2.1
            - RV2.2
```